addons/l10n_jp/data/account_tax_template_data.xml

```diff
@@ -303,32 +303,32 @@
         <field name="invoice_repartition_line_ids" eval="[(5, 0, 0),
             (0,0, {
                 'repartition_type': 'base',
-                'plus_report_expression_ids': [ref('l10n_jp.tax_report_comp_basis_purchases_taxable_10_tag')],
+                'plus_report_expression_ids': [ref('l10n_jp.l10n_jp_tax_report_comp_basis_purchases_taxable_10_tag')],
             }),
             (0,0, {
                 'repartition_type': 'tax',
-                'account_id': ref('A11809'),
-                'plus_report_expression_ids': [ref('l10n_jp.tax_report_to_pay_temp_pmt_susp_cons_10_tag')],
+                'account_id': ref('l10n_jp_123200'),
+                'plus_report_expression_ids': [ref('l10n_jp.l10n_jp_tax_report_to_pay_temp_pmt_susp_cons_10_tag')],
             }),
         ]"/>
         <field name="refund_repartition_line_ids" eval="[(5, 0, 0),
             (0,0, {
                 'repartition_type': 'base',
-                'minus_report_expression_ids': [ref('l10n_jp.tax_report_comp_basis_purchases_taxable_10_tag')],
+                'minus_report_expression_ids': [ref('l10n_jp.l10n_jp_tax_report_comp_basis_purchases_taxable_10_tag')],
             }),
             (0,0, {
                 'repartition_type': 'tax',
-                'account_id': ref('A11809'),
-                'minus_report_expression_ids': [ref('l10n_jp.tax_report_to_pay_temp_pmt_susp_cons_10_tag')],
+                'account_id': ref('l10n_jp_123200'),
+                'minus_report_expression_ids': [ref('l10n_jp.l10n_jp_tax_report_to_pay_temp_pmt_susp_cons_10_tag')],
             }),
         ]"/>
-        <field name="tax_group_id" ref="tax_group_10"/>
+        <field name="tax_group_id" ref="l10n_jp_tax_group_10"/>
     </record>
-    <record id="tax_out_im" model="account.tax.template">
```

Comment:
I think by default an importation is a purchase + I would use 'imp' instead of 'im'